workflows_mgr.py

```diff
@@ -109,8 +109,12 @@ def spawn_workflow(self):
             if w_id in scanflows:
                 if (info['host'] == scanflows[w_id]['host'] and
                         info['port'] == scanflows[w_id]['port']):
+                    client = scanflows[w_id]['req_client']
+                    client.socket.close()
                     scanflows.pop(w_id)
                 continue
+            client = self.workflows[w_id]['req_client']
+            client.socket.close()
```

Comment:
I guess these `.close()` methods are safe? Worth wrapping them in `try ... except IOError`?

Reply:
Suppressing the `IOError` now :+1: